src/components/TodoComponents/TodoList.js

```diff
@@ -1,2 +1,15 @@
 // your components will all go in this `component` directory.
 // feel free to change this component.js into TodoList.js
+import React from 'react';
+import Todo from './Todo';
+
+ const TodoList = ({list}) => {
+        const data = list.map((todo, index)=>{
+        return (
+            <Todo key={index} item={todo}/>
+        )
+        })
+        return <div>{data}</div>
```

Comment:
this is a fine way to do this. you could also drop the map function directly into the JSX without declaring a variable first

```
const TodoList = ({list}) => {
    return (
        <div>
        {list.map((todo, index)=>{
            return <Todo key={index} item={todo}/>
         })}
        </div>
      )
}
```